common.go

```diff
@@ -25,39 +25,20 @@ var DefaultStrategies = []Strategy{
 }
 
 // GetLanguage applies a sequence of strategies based on the given filename and content
-// to find out the most probably language to return.
-func GetLanguage(filename string, content []byte) string {
-	var languages []string
-	candidates := []string{}
-	for _, strategy := range DefaultStrategies {
-		languages = strategy(filename, content, candidates)
-		if len(languages) == 1 {
-			return languages[0]
-		}
-
-		if len(languages) > 0 {
-			candidates = append(candidates, languages...)
-		}
-	}
-
-	return firstLanguage(languages)
+// to find out the most probably language to return. If there are more than one possibles languages
+// it returns safe to false.
+func GetLanguage(filename string, content []byte) (language string, safe bool) {
```

Comment:
Sorry, I think I didn't express it properly: maintain previous signature, we don't want that `safe` boolean, what we want is adding `GetLanguages` with the language list.